srcpkgs/tickr/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'tickr'
+pkgname=tickr
+version=0.6.4
+revision=1
+build_style=gnu-configure
+hostmakedepends="gtk+-devel libxml2-devel"
+makedepends="pkg-config"
```

Comment:
You mixed hostmakedepends and makedepends up. It should be the other way around.
```
hostmakedepends="pkg-config"
makedepends="gtk+-devel libxml2-devel"
```

Reply:
will fix